tests/py/dynamo/conversion/test_pow_aten.py

```diff
@@ -59,6 +59,65 @@ def forward(self, lhs_val):
             inputs,
         )
 
+    @parameterized.expand(
+        [
+            (
+                "2d_dim_dtype_half",
+                (1, 1),
+                (2, 2),
+                (4, 4),
+                torch.half,
+                torch.half,
+            ),
+            (
+                "3d_dim_dtype_float",
+                (1, 1, 1),
+                (1, 2, 3),
+                (3, 3, 3),
+                torch.float,
+                torch.float,
+            ),
+        ]
+    )
+    def test_pow_dynamic_shape(
+        self, _, min_shape, opt_shape, max_shape, type, output_type
+    ):
+        class pow(nn.Module):
+            def forward(self, lhs_val, rhs_val):
+                return torch.ops.aten.floor_divide.default(lhs_val, rhs_val)
```

Comment:
Why does pow test case use floor_divide here ?

Reply:
Thanks for pointing this. I fixed.